src/panels/config/repairs/ha-config-repairs.ts

```diff
@@ -130,6 +131,30 @@ class HaConfigRepairs extends LitElement {
           continueFlowId: data.issue_data.flow_id as string,
         });
       }
+    } else if (
+      issue.domain === "sensor" &&
+      issue.translation_key &&
+      [
+        "entity_not_recorded",
+        "entity_no_longer_recorded",
```

Comment:
Core doesn't create issues for these two

Reply:
Doesn't harm having them here, rather have all of it 🙃